docs/medical-api/more-info/webhooks.mdx

```diff
@@ -1,22 +1,31 @@
 ---
-title: "Webhooks"
-description: "Receive status updates as soon as data becomes available after initiating a document query."
+title: "Webhooks - Medical API"
+description: "Receive status updates as soon as data becomes available."
 ---
 
 ## Overview
 
-When using the Medical API, Metriport will send Webhook messages containing [patient document data](#patient-data-message) to your app, as soon
-as the data becomes available after initiating a [document query](/medical-api/api-reference/document/start-document-query).
+When using the Medical API, Metriport will send Webhook messages to your app as soon as data becomes available.
 
-You can expect to receive updates in the following scenarios:
-
-1. When queried documents have completed downloading (`type` for a patient in the message will be `document-download`, and at this point you'll be able to [download the raw files](/medical-api/api-reference/document/get-document));
-1. Then, if the downloaded documents contained C-CDA/XML files, when the conversion to FHIR has completed (`type` for a patient in the message will be `document-conversion`, and at this point you'll be able to [query for FHIR resrouces](/medical-api/api-reference/fhir/get-patient-consolidated)).
+This is the usual flow to obtain Patient data:
+1. You trigger a [document query](/medical-api/api-reference/document/start-document-query) to retrieve a Patient's medical documents from HIEs;
+   - XML C-CDA documents will be converted and stored on Metriport's FHIR storage;
```

Comment:
nit: in some places we call it `FHIR storage`, and in others we say `FHIR repository` - let's pick one and be consistent